document/js-api/index.bs

```diff
@@ -687,6 +695,90 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
     1. Return undefined.
 </div>
 
+<h3 id="globals">Globals</h3>
+
+<pre class="idl">
+typedef (long or float or double) GlobalValue;
+
+dictionary GlobalDescriptor {
+  required USVString type;
+  GlobalValue value;
+  boolean mutable;
+};
+
+[LegacyNamespace=WebAssembly, Constructor(GlobalDescriptor descriptor), Exposed=(Window,Worker,Worklet)]
+interface Global {
+  GlobalValue valueOf();
+  attribute GlobalValue value;
+};
+</pre>
+
+<div>
+A {{Global}} object represents a single [=global instance=]
+which can be simultaneously referenced by multiple {{Instance}} objects. Each
+{{Global}} object has one internal slot:
+
+    * \[[Global]] : a [=global address=]
+</div>
+
+<div algorithm>
+    To <dfn>create a global object</dfn> from a [=global address=] |globaladdr|, perform the following steps:
+    1. Let |map| be the current [=agent=]'s associated [=Global object cache=].
+    1. If |map|[|globaladdr|] [=map/exists=],
+        1. Return |map|[|globaladdr|].
+    1. Let |global| be a new {{Global}} instance with \[[Global]] set to |globaladdr|.
+    1. [=map/Set=] |map|[|globaladdr|] to |global|.
+    1. Return |global|.
+</div>
+
+<div algorithm>
+    To <dfn>ToValueType</dfn>(|s|), perform the following steps:
+    1. If |s| equals "i32", return [=𝗂𝟥𝟤=].
+    1. If |s| equals "f32", return [=𝖿𝟥𝟤=].
+    1. If |s| equals "f64", return [=𝖿𝟨𝟦=].
+    1. Otherwise, throw a {{TypeError}} exception.
+</div>
+
+<div algorithm>
+    The <dfn constructor for="Global">Global(descriptor)</dfn> constructor, when invoked, performs the following steps:
+    1. If |descriptor|["value"] is [=present=], let |v| be |descriptor|["value"]; otherwise, let |v| be 0.
+    1. If |descriptor|["mutable"] is [=present=], let |mutable| be |descriptor|["mutable"]; otherwise, let |mutable| be false.
+    1. Let |type| be [=ToValueType=](|descriptor|["type"]).
+    1. Let |value| be [=ToWebAssemblyValue=](|v|, |type|).
+    1. If |mutable| is true, let |globaltype| be [=var=] |type|; otherwise, let |globaltype| be [=const=] |type|.
+    1. Let |store| be the current agent's [=associated store=].
```

Comment:
Nit: I think this should be `[=surrounding agent=]` here and elsewhere. Maybe a preexisting problem with the rest of the spec; haven't checked.

Reply:
Yeah, it's not used anywhere else currently. I'll open a PR to fix the spec upstream.